LoopKit/InsulinKit/PumpEventType.swift

```diff
@@ -50,6 +148,12 @@ extension PumpEventType {
             return 8
         case .basal:
             return 9
+        case .replaceComponent(componentType: .reservoir):
+            return 4
```

Comment:
This can cause identity conflicts if all of these types have the same ID of 4. I'd make sure these are unique.

Reply:
Agree, per offline discussion will set these to 10, 11, 12 as placeholders until there is further clarification on the order.